packages/core/src/App/Containers/AccountSignupModal/account-signup-modal.jsx

```diff
@@ -1,9 +1,8 @@
 import classNames        from 'classnames';
 import {
-    Autocomplete,
+    PasswordInput,
```

Comment:
Please reorder these imports for `Dialog` to be before `PasswordInput`. This is because a current build configuration throws warnings for CSS import order of these components and will be addressed in a separate PR.